code/game/objects/items/weapons/material/twohanded.dm

```diff
@@ -540,8 +540,8 @@
 	name = "halberd"
 	desc = "A sharp axe mounted on the top of a long spear."
 	force = 10
-	unwielded_force_divisor = 0.4
-	force_divisor = 0.6
+	unwielded_force_divisor = 0.2

```

Comment:
You can't make halberds, you can only get them via the merchant and adminbus. No reason to nerf them.

Reply:
Merchant/adminbus or not, those values still make a plasteel/diamond halberd stronger than an energy glaive.  28 per hit damage while dual wielded is still plenty to be going along with for a plasteel halberd (or 35 for a diamond one).

Reply:
You can't make halberds with any other material, they are always steel.

Reply:
Ah, that makes things different.  Hokay.

Reply:
Also, not sure how this nerfs pikes if it is touching halberds.

Reply:
Halberds are a child of pikes, so it doesn't affect pikes themselves at all.

Reply:
You can only make the regular pikes, not halberds.